### ISSUE TITLE
OAI - work to include non DC terms in OAI feed has been contributed back to hyku.

### ISSUE THREAD

**Issue by crisr15** [scientist-softserv/palni-palci#425]
# Summary

CURRENTLY BLOCKED BY #425

Work from Epic #403 has been contributed back to hyku.  

# Acceptance Criteria

- [ ] Work from Epic #403 has been contributed back to hyku.  
- [ ] Reconcile fields (remove PALs-specific fields, add any missing Hyku default fields) 
- [ ] Make a ticket in Hyku if any fields aren't displaying correctly (e.g. `based_near`) 
  - See https://github.com/scientist-softserv/palni-palci/issues/421#issuecomment-1454312045 

#  Testing Instructions

1. Spin up Hyku locally 
2. Create a couple "full" works (i.e. all metadata fields are filled out) 
3. Navigate to OAI feed (`/catalog/oai`) 
4. Verify all public fields display in the feed using the `oai_hyku` prefix 

Part of Epic #403

**Comment by bkiahstroud** [comment]
@sephirothkod assuming https://github.com/scientist-softserv/palni-palci/issues/423 passes QA, this ticket is no longer blocked

**Comment by crisr15** [comment]
Merged into hyku on ticket https://github.com/samvera/hyku/pull/1934.
